src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -1442,8 +1429,6 @@ public SavePreferences loadForExportFromPreferences() {
         if (!saveInOriginalOrder) {
             if (this.getBoolean(JabRefPreferences.EXPORT_IN_SPECIFIED_ORDER)) {
                 saveOrder = this.loadExportSaveOrder();
-            } else {
-                saveOrder = this.loadTableSaveOrder();
```

Comment:
I suspect this is still needed to allow the user to export entries in the current sort order.

Reply:
I still think the export sort order should be different from the table sort order and I will remove table sort order. But I currently see no easy way to convert the table column sort order to our schema

Reply:
I'm fine with removing the "export according to table sorting" feature since it may quickly lead to unexpected exports (although I'm pretty confident that some users rely on this feature). But then please add a changelog entry and check if the documentation needs to be adopted accordingly.

Reply:
I quickly had an idea on how to re implement this which works.